libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -351,64 +351,64 @@ AC_PosControl::AC_PosControl(AP_AHRS_View& ahrs, const AP_Motors& motors, AC_Att
 /// 3D position shaper
 ///
 
-// Sets a new NEU position target in meters and computes a jerk-limited trajectory.
+// Sets a new NED position target in meters and computes a jerk-limited trajectory.
 // Updates internal acceleration commands using a smooth kinematic path constrained
 // by configured acceleration and jerk limits. Terrain margin is used to constrain
 // horizontal velocity to avoid vertical buffer violation.
-void AC_PosControl::input_pos_NEU_m(const Vector3p& pos_neu_m, float pos_terrain_target_u_m, float terrain_buffer_m)
+void AC_PosControl::input_pos_NED_m(const Vector3p& pos_ned_m, float pos_terrain_target_d_m, float terrain_buffer_m)
```

Comment:
while it's an existing problem, I think we should add a target explaining the last two arguments.  "terrain margin" is mentioned in the comments but it's been renamed at some point to "terrain buffer"

Reply:
Hi, yes, just a change to the comment above the function